sky/packages/sky/lib/gestures/drag.dart

```diff
@@ -149,8 +149,8 @@ class PanGestureRecognizer extends _DragGestureRecognizer<sky.Offset> {
 
   sky.Offset get _initialPendingDragDelta => sky.Offset.zero;
   // Notice that we negate dy because scroll offsets go in the opposite direction.
-  sky.Offset _getDragDelta(sky.PointerEvent event) => new sky.Offset(event.dx, -event.dy);
+  sky.Offset _getDragDelta(sky.PointerEvent event) => new sky.Offset(-event.dx, -event.dy);
   bool get _hasSufficientPendingDragDeltaToAccept {
-    return _pendingDragDelta.dx.abs() > kTouchSlop || _pendingDragDelta.dy.abs() > kTouchSlop;
+    return _pendingDragDelta.distance > kPanSlop;
```

Comment:
no, this is wrong. this changes the slop shape from a square to a circle. It needs to be a square.